src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -44,12 +37,10 @@ import { ACADEMIC_GRANTS_THANK_YOU_PAGE_URL, TOAST_OPTIONS } from '../../constan
 
 import { AcademicGrantsFormData, ApplyingAs, GrantsReferralSource } from '../../types';
 
-const MotionBox = motion<BoxProps>(Box);
-const MotionButton = motion<ButtonProps>(Button);
-
 export const AcademicGrantsForm: FC = () => {
   const router = useRouter();
   const toast = useToast();
+  const [isSubmitting, setIsSubmitting] = useState(false);
```

Comment:
`useForm` gives you this state already https://react-hook-form.com/api/useform/formstate

You should pass a Promise (turn `onSubmit` to async) to `handleSubmit` instead.

Reply:
much cleaner! code updated